lib/constants/state.js

```diff
@@ -0,0 +1,13 @@
+const state = {
```

Comment:
It's not clear from the name what this variable is about. Please, make naming more specific.

Reply:
tvOSElementState?

Reply:
How about `elementState`?

Reply:
Also, we should keep it consistent with other constants. E.g. there is `ELEMENT_PROPS` (not `elementProps`). Please update new constants to fit that naming convention